k_bad: 0
particles_num: 1000
sense_noise: 0.00075
distance_noise: 0.002
angle_noise: 0.0035
min_intens: 3000
min_point_per_beacon: 3
max_dist: 3700
k_angle: 0
beac_dist_thresh: 0.95
num_is_near_thresh: 0.1
distance_noise_1_beacon: 0.0133
angle_noise_1_beacon: 0.02
sigma_phi: 0.06
min_sin: 0.4
min_cost_function: 0
sigma_r: 0.045
num_seeing_beacons: 3
particles_num_from_measurement_model: 5